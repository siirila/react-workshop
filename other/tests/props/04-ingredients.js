import React from 'react'
import {render} from 'enzyme'
const Example = require(`../../../src/props/${process.env.TEST_DIR ||
  'exercises'}/04-ingredients`).Example

test('Renders "The expected list of ingredients', () => {
  expect(render(<Example />)).toMatchSnapshot()
})
