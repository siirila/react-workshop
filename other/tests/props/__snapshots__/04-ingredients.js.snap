// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "The expected list of ingredients 1`] = `
<div>
  <h3>
    Blueberry Oatmeal Squares
  </h3>
  <ul
    style="list-style-type:none"
  >
    <span
      style="font-weight:bold"
    >
      Ingredients
    </span>
    <li>
      2-1/2 cups - rolled oats, (not instant)
    </li>
    <li>
      1-1/4 cups - all-purpose flour
    </li>
    <li>
      1 tbsp - grated orange rind
    </li>
    <li>
      1/4 tsp - salt
    </li>
    <li>
      1 cup - cold butter, cubed
    </li>
    <li>
      3/4 cup - packed brown sugar
    </li>
    <li>
      3 cups - fresh blueberries
    </li>
    <li>
      1/2 cup - granulated sugar
    </li>
    <li>
      1/3 cup - orange juice
    </li>
    <li>
      4 tsp - cornstarch
    </li>
  </ul>
</div>
`;
